packages/wct-mocha/src/index.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+/**
+ * This file is the entry point into `web-component-tester`'s browser client.
+ */
+// Registers a bunch of globals:
```

Comment:
Looks like these aren't globals anymore?

Reply:
At the bottom of the helpers.ts file it throws a bunch of stuff on window

Reply:
```ts
declare global {
  interface Window {
    safeStep: typeof safeStep;
    testImmediate: typeof testImmediate;
    testImmediateAsync: typeof testImmediateAsync;
    flush: typeof flush;
    animationFrameFlush: typeof animationFrameFlush;
    asyncPlatformFlush: typeof asyncPlatformFlush;
    waitFor: typeof waitFor;
  }
}

window.safeStep = safeStep;
window.testImmediate = testImmediate;
window.testImmediateAsync = testImmediateAsync;
window.flush = flush;
window.animationFrameFlush = animationFrameFlush;
window.asyncPlatformFlush = asyncPlatformFlush;
window.waitFor = waitFor;
```

I assume that's what the comment was referring to.